frontend/src/components/EditableSelect.module.scss

```diff
@@ -9,6 +9,15 @@
   margin-top: 0;
 }
 
+.editButtonDiv {
+  margin-top: 0;
```

Comment:
I did not see this affecting the styles (?). If that's correct, the line can be removed